CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ include(cmake/xcode.cmake)
 CMAKE_DEPENDENT_OPTION(KTX_FEATURE_TOOLS "Create KTX tools" ON "NOT IOS;NOT ANDROID;NOT EMSCRIPTEN" OFF)
 option( KTX_FEATURE_DOC "Create KTX documentation" OFF )
 option( KTX_FEATURE_LOADTEST_APPS "Create load tests apps that load and display various KTX textures" OFF )
+option( KTX_FEATURE_FORCE_STATIC "Always create static libraries" OFF )
```

Comment:
Perhaps it would be better for the static library option to always appear and be set and unchangeable on iOS and Emscripten, unset and changeable everywhere else. This would show people that the library on those platforms is static, something hidden at present. If we do this the pop-up help should explain why it may be unchangeable.

What do you think @atteneder?

Reply:
Plural (libraries), because ktx_read is added with this changes making it two libraries.
I've updated this to an option, that's not available on iOS/Emscripten.

I can see the benefit of having the static option always there, but I don't think you can make it unchangeable (as in greyed-out in CMake GUI). A fallback could be to correct invalid configurations (e.g. OFF on iOS) by throwing a warning or an error. I'd prefer an error (warnings may not be read and can lead to confusion).

I have no strong opinion on whether the static option should be present on all platforms or not.

Reply:
If you can't disable the option in the GUI then its better to just not have it  on iOS and Emscripten.